vumi/transports/dmark/dmark_ussd.py

```diff
@@ -19,6 +19,10 @@ class DmarkUssdTransportConfig(HttpRpcTransport.CONFIG_CLASS):
         " expires.",
         default=600, static=True)
 
+    fix_to_addr = ConfigBool(
+        "Whether or not to ensure that the to_addr is always starting with a "
+        "* and ending with a #", default=False, static=True)
```

Comment:
it doesn't change per message received, it's a config thing on their side so I would think it is static? That way I can use the `get_static_config()` helper and it's still cheap to do so.